server/network.go

```diff
@@ -46,6 +46,8 @@ type localNetwork struct {
 	stopOnce sync.Once
 
 	subnets []string
+	// map from subnet ID to participating node info
+	subnetParticipants map[string][]*rpcpb.NodeInfo
```

Comment:
would rather map subnet ID to node name, so as to not repeat node info in status, and let the user find out more information on the node